docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,234 @@
+= Integrating CMOS with Existing Monitoring Stacks
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+You can integrate CMOS with your existing Prometheus, Alertmanager, and Grafana monitoring system. 
+This tutorial documents how to configure each component in order to achieve this.
+
+include::partial$tutorial.adoc[]
+
+== Pre-requisites
+
+The environment you wish to integrate CMOS into must have:
+
+. At least one **Couchbase Server** node;
+. **Prometheus**, with **Alertmanager** installed;
+. **Grafana**, configured to use the Prometheus server as a Data Source;
+
+==== Optional ====
+
+- **Docker**, if using the Cluster Monitor's Docker image instead of the binary.
```

Comment:
If we're only recommending docker for CBMM, should we make it a full requirement?